ingestion/tests/unit/topology/pipeline/test_dagster.py

```diff
@@ -51,9 +51,9 @@
                 "type": "Dagster",
                 "hostPort": "http://localhost:3000",
                 "dbConnection": {
-                    "username": "dagster_user",
-                    "password": "dagter_pass",
-                    "databaseSchema": "dagster_db",
+                    "username": "user",
+                    "password": "password",
+                    "databaseSchema": "database_db",
```

Comment:
is `dbConnection` field still required? can we remove this from json schema itself?